ql/src/test/results/clientpositive/llap/avrotblsjoin.q.out

```diff
@@ -71,9 +71,9 @@ POSTHOOK: Input: _dummy_database@_dummy_table
 POSTHOOK: Output: default@table1_1
 POSTHOOK: Lineage: table1_1.col1 SCRIPT []
 POSTHOOK: Lineage: table1_1.col2 SCRIPT []
-WARNING: Comparing a bigint and a string may result in a loss of precision.
-WARNING: Comparing a bigint and a string may result in a loss of precision.
-WARNING: Comparing a bigint and a string may result in a loss of precision.
+WARNING: Comparing string and bigint may result in loss of information.
```

Comment:
Maybe `a loss of precision` is more user friendly compared to `information`.

Reply:
I changed the message `to loss of information` since precision may not make sense for certain types such as string. I don't have a strong preference though so I can restore the old message.